In musl libc these macros are defined in the deprecated
include file sys/cdefs.h

--- a/include/libcgroup.h	2014-01-13 15:05:56.000000000 +0100
+++ b/include/libcgroup.h	2015-10-25 14:03:32.921207668 +0100
@@ -16,6 +16,16 @@
 #ifndef _LIBCGROUP_H
 #define _LIBCGROUP_H

+#if !defined(__GLIBC__)
+#ifdef  __cplusplus
+# define __BEGIN_DECLS	extern "C" {
+# define __END_DECLS	}
+#else
+# define __BEGIN_DECLS
+# define __END_DECLS
+#endif
+#endif /* !defined(__GLIBC__) */
+
 #define _LIBCGROUP_H_INSIDE
 
 #include <libcgroup/error.h>
